enterprise/admin/forms.py

```diff
@@ -501,3 +502,43 @@ class Meta:
             'decrypted_password': forms.widgets.PasswordInput(),
             'decrypted_sftp_password': forms.widgets.PasswordInput(),
         }
+
+
+class TransmitEnterpriseCoursesForm(forms.Form):
+    """
+    Form to transmit courses metadata for enterprise customers.
+    """
+    catalog_user = forms.CharField(
```

Comment:
catalog_user or enterprise worker user?

Reply:
`Enterprise channel worker` user (Catalog user) and `enterprise worker` user are two different user setup for enterprise service. Enterprise channel worker is used to push courses metadata for enterprise integrated channels.
https://s3.amazonaws.com/uploads.hipchat.com/26537/688531/7etSDD6TWexLuZP/worker_enterprise.png